python/pyspark/ml/tuning.py

```diff
@@ -883,7 +895,7 @@ class CrossValidatorModel(Model, _CrossValidatorParams, MLReadable, MLWritable):
     .. versionadded:: 1.4.0
```

Comment:
can we add some docs for parameters since we're here? (see also https://numpydoc.readthedocs.io/en/latest/format.html#parameters)